module/Olcs/src/Form/Model/Fieldset/Message/Create.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Olcs\Form\Model\Fieldset\Message;
+
+use Common\Form\Elements\InputFilters\ActionButton;
+use Laminas\Form\Annotation as Form;
+use Laminas\Form\Element\Textarea;
+
+class Create
+{
+    /**
+     * @Form\Options({
+     *     "label": "messaging.subject",
+     *     "empty_option": "Please select",
+     *     "disable_inarray_validator": false,
+     *     "service_name": "Common\Service\Data\MessagingSubject"
```

Comment:
C: If it works here, switch to `\Common\Service\Data\MessagingSubject::class` (and further down, same thing)